### PR TITLE
Fix Approved Premises config

### PR DESCRIPTION
**Pull request by pezholio**
We had some names and teams set incorrectly, and the ecr config didn't have a repo set, which meant we couldn't deploy.

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/ecr.tf [hunk 1]
```diff
@@ -5,27 +5,12 @@
...
-  # Uncomment and provide repository names to create github actions secrets
-  # containing the ECR name, AWS access key, and AWS secret key, for use in
-  # github actions CI/CD pipelines
-  # github_repositories = ["my-repo"]
-
-  # list of github environments, to create the ECR secrets as environment secrets
-  # https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-secrets
-  # github_environments = ["my-environment"]
-
+  github_repositories = [var.namespace]
```
Comment (jackstockley89) [inline namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/ecr.tf:13]: Are your namespace and github name the same

File: namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/variables.tf [hunk 2]
```diff
@@ -10,11 +10,11 @@ variable "kubernetes_cluster" {
 
 variable "application" {
   description = "Name of Application you are deploying"
-  default     = "Gov.UK Prototype Kit"
+  default     = "Approved Premises Prototype"
 }
 
 variable "namespace" {
-  default = "approved-premises-prototype"
+  default = "approved-premises-team"
```
Comment (jackstockley89) [inline namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/variables.tf:17]: This should still be "approved-premises-prototype"
The variable that needed to be change was 

variable "team_name" {
  description = "The name of your development team"
  default     = "approved-premises-prototype" > "approved-premises-team"
}

Comment (pezholio) [reply namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/variables.tf:17]: Gotcha. Will fix 👍

File: namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/ecr.tf (hunk 1 above)
Comment (pezholio) [reply namespaces/live.cloud-platform.service.justice.gov.uk/approved-premises-prototype/resources/ecr.tf:13]: Yep, both `approved-premises-prototype`

**Review by jackstockley89 — APPROVED** [review]

**Pull request merged**